### PR TITLE
NGINX Ingress Controller and Resources

### PR DESCRIPTION
**Pull request by aluisribeiro**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
We don't have a Ingress Controller

* **What is the new behavior (if this is a feature change)?**
NGINX Ingress Controller

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
Yes, this PR is connected to dojot/dojot#1600

* **Other information**:

### CODE REVIEW

File: roles/nginx-ingress/templates/ingress-resources/http.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  namespace: {{ dojot_namespace }}
+spec:
+  rules:
+  - host: {{ dojot_domain_name }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: kong
+          servicePort: 8000
```
Comment (eduardogmisiuk) [inline roles/nginx-ingress/templates/ingress-resources/http.yaml.j2:14]: You can use the `dojot_internal_http_port` variable in here.

**Review by marianoleonardo — DISMISSED** [review]
LGTM!

**Comment by mprevide** [comment]
You can change this to release/v0.5.1 when it is created or to dev

**Comment by andersonluisribeiro** [comment]
At this time, we will not use the ingress controller.

**Pull request closed without merging**